rasa_nlu/persistor.py

```diff
@@ -67,6 +67,11 @@ def list_models(self, project):
 
         raise NotImplementedError
 
+    def list_projects(self):
```

Comment:
needs a short descriptions so that implementors know what to do

Reply:
Yes, you are right. I will add a brief doc-string to this method.